### PR TITLE
Added quickstart verification step

### PR DESCRIPTION
**Pull request by garanj**
Added a verification step that provides a link to the user's newly-created CRMint app, and has them browse to it to confirm correct installation.

### CODE REVIEW

**Comment by garanj** [comment]
Having now seen the CRMint App, I've added some placeholders in the guide for explaining core concepts like pipelines, jobs, workers etc.

**Comment by garanj** [comment]
- Fixed a typo in Quickstart
- Added a verfication launch in Quickstart
- Placeholder for the Concepts guide (WIP offline)
- Workers reference

File: docs/_site/js/buttons.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+(function ($) {
```
Comment (Slony) [inline docs/_site/js/buttons.js]: Can you, please, add license info here?

File: docs/_site/js/common.js [hunk 2]
```diff
@@ -0,0 +1,368 @@
+// Youtube Player API
```
Comment (Slony) [inline docs/_site/js/common.js]: Can you, please, add license info here?

File: docs/_site/js/jquery.collapsible.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
+/*!
```
Comment (Slony) [inline docs/_site/js/jquery.collapsible.js]: Can you, please, add license info here?

File: docs/_site/js/jquery.form.min.js [hunk 4]
```diff
@@ -0,0 +1 @@
... e(t).val()},e.fn.clearForm=function(t){return this.each(function(){e("input,select,textarea",this).clearFields(t)})},e.fn.clearFields=e.fn.clearInputs=function(t){var r=/^(?:color|date|datetime|email|month|number|password|range|search|tel|text|time|url|week)$/i;return this.each(function(){var a=this.type,n=this.tagName.toLowerCase();r.test(a)||"textarea"==n?this.value="":"checkbox"==a||"radio"==a?this.checked=!1:"select"==n?this.selectedIndex=-1:"file"==a?/MSIE/.test(navigator.userAgent)?e(this).replaceWith(e(this).clone(!0)):e(this).val(""):t&&(t===!0&&/hidden/.test(a)||"string"==typeof t&&e(this).is(t))&&(this.value="")})},e.fn.resetForm=function(){return this.each(function(){("function"==typeof this.reset||"object"==typeof this.reset&&!this.reset.nodeType)&&this.reset()})},e.fn.enable=function(e){return void 0===e&&(e=!0),this.each(function(){this.disabled=!e})},e.fn.selected=function(t){return void 0===t&&(t=!0),this.each(function(){var r=this.type;if("checkbox"==r||"radio"==r)this.checked=t;else if("option"==this.tagName.toLowerCase()){var a=e(this).parent("select");t&&a[0]&&"select-one"==a[0].type&&a.find("option").selected(!1),this.selected=t}})},e.fn.ajaxSubmit.debug=!1});
```
Comment (Slony) [inline docs/_site/js/jquery.form.min.js]: Can you, please, add the license info here.

File: docs/_site/js/jquery.visible.min.js [hunk 5]
```diff
@@ -0,0 +1 @@
+!function(t){var i=t(window);t.fn.visible=function(t,e,o){if(!(this.length<1)){var r=this.length>1?this.eq(0):this,n=r.get(0),f=i.width(),h=i.height(),o=o?o:"both",l=e===!0?n.offsetWidth*n.offsetHeight:!0;if("function"==typeof n.getBoundingClientRect){var g=n.getBoundingClientRect(),u=g.top>=0&&g.top<h,s=g.bottom>0&&g.bottom<=h,c=g.left>=0&&g.left<f,a=g.right>0&&g.right<=f,v=t?u||s:u&&s,b=t?c||a:c&&a;if("both"===o)return l&&v&&b;if("vertical"===o)return l&&v;if("horizontal"===o)return l&&b}else{var d=i.scrollTop(),p=d+h,w=i.scrollLeft(),m=w+f,y=r.offset(),z=y.top,B=z+r.height(),C=y.left,R=C+r.width(),j=t===!0?B:z,q=t===!0?z:B,H=t===!0?R:C,L=t===!0?C:R;if("both"===o)return!!l&&p>=q&&j>=d&&m>=L&&H>=w;if("vertical"===o)return!!l&&p>=q&&j>=d;if("horizontal"===o)return!!l&&m>=L&&H>=w}}}}(jQuery);
```
Comment (Slony) [inline docs/_site/js/jquery.visible.min.js]: Can you, please, add the license info here.

File: docs/_site/js/mc-validate.js [hunk 6]
```diff
@@ -0,0 +1,202 @@
+$(document).ready(function() {
```
Comment (Slony) [inline docs/_site/js/mc-validate.js]: Can you, please, add license info here?

File: docs/_site/js/slick.min.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+|___/_|_|\___|_|\_(_)/ |___/
+                   |__/
+
+ Version: 1.6.0
+  Author: Ken Wheeler
+ Website: http://kenwheeler.github.io
+    Docs: http://kenwheeler.github.io/slick
+    Repo: http://github.com/kenwheeler/slick
+  Issues: http://github.com/kenwheeler/slick/issues
+
```
Comment (Slony) [inline docs/_site/js/slick.min.js]: Please, add the license here:

```
License:
Copyright (c) 2017 Ken Wheeler

Licensed under the MIT license.

Free as in Bacon.
```

File: docs/_site/js/slick.min.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+ */
```
Comment (Slony) [inline docs/_site/js/slick.min.js]: Can you, please, add the license info here.

File: docs/js/common.js [hunk 9]
```diff
@@ -11,6 +11,7 @@ firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
 $('#project-id').on('input',  function() {
```
Comment (Slony) [inline docs/js/common.js]: Can you, please, add license info to this file?

**Pull request merged**